### ISSUE TITLE
CLIP Guided StableDiffusion V2

### ISSUE THREAD

**Issue by dblunk88** [huggingface/diffusers#1404]
### Describe the bug

When trying to use the CLIP guided pipeline with Stable Diffusion V2, it errors out

### Reproduction

See above

### Logs

```shell
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-254971518fc8> in <module>
     13                                                     custom_pipeline="clip_guided_stable_diffusion",
     14                                                     clip_model=clip_model,
---> 15                                                     feature_extractor=feature_extractor
     16                                                     ).to('cuda')
     17 guided_pipeline.unfreeze_vae()

1 frames
/usr/local/lib/python3.7/dist-packages/diffusers/pipeline_utils.py in from_pretrained(cls, pretrained_model_name_or_path, **kwargs)
    662 
    663         # 5. Instantiate the pipeline
--> 664         model = pipeline_class(**init_kwargs)
    665         return model
    666 

~/.cache/huggingface/modules/diffusers_modules/git/clip_guided_stable_diffusion.py in __init__(self, vae, text_encoder, clip_model, tokenizer, unet, scheduler, feature_extractor)
     78         )
     79 
---> 80         self.normalize = transforms.Normalize(mean=feature_extractor.image_mean, std=feature_extractor.image_std)
     81         self.make_cutouts = MakeCutouts(feature_extractor.size)
     82 

AttributeError: 'NoneType' object has no attribute 'image_mean'
```

### System Info

Colab

**Comment by averad** [comment]
model_index.json
```
  "feature_extractor": [
    null,
    null
  ],
```

**Comment by pcuenca** [comment]
Fixed in #1414.
